code/game/gamemodes/meteor/meteors.dm

```diff
@@ -319,31 +356,102 @@ GLOBAL_LIST_INIT(meteorsC, list(/obj/effect/meteor/dust)) //for space dust event
 	if(!isspaceturf(T))
 		new /obj/effect/decal/cleanable/xenoblood(T)
 
-//Station buster Tunguska
-/obj/effect/meteor/tunguska
-	name = "tunguska meteor"
-	icon_state = "flaming"
-	desc = "Your life briefly passes before your eyes the moment you lay them on this monstrosity."
-	hits = 30
+//Blobby-Meaty Ore
+/obj/effect/meteor/meaty/mouldy
+	name = "mouldy ore"
+	desc = "Ewww."
+	color = "#654321"
+	meteorsound = 'sound/effects/splat.ogg'
+	meteordrop = list(/obj/item/reagent_containers/food/snacks/meat/slab/human/mutant/zombie, /obj/item/organ/tongue/zombie)
+	threat = 10
+	var/spawn_max = 4
+	var/blobbo_chance = 25
+
+/obj/effect/meteor/meaty/mouldy/meteor_effect()
+	..()
+	if(prob(blobbo_chance))
+		new /mob/living/simple_animal/hostile/blob/blobbernaut/independent(src.loc, src)
+	else
+		for(var/i in 1 to spawn_max)
+			var/mob/living/simple_animal/hostile/blob/blobspore/BS = new /mob/living/simple_animal/hostile/blob/blobspore(src.loc, src)
+			for(var/j = 1, j <= rand(1, 3), j++)
+				step(BS, pick(NORTH,SOUTH,EAST,WEST))
+
+//Bone-Meaty Ore
+/obj/effect/meteor/meaty/bone
+	name = "boney ore"
+	desc = "This thing sure drank its calcium."
+	icon = 'icons/obj/statuelarge.dmi'
+	icon_state = "skull"
+	meteordrop = list(/obj/item/stack/sheet/bone, /obj/item/organ/tongue/bone, /obj/item/reagent_containers/food/snacks/meat/slab/human/mutant/skeleton)
+	dropamt = 8 //No bone gib spawner, so lots of bones instead. Besides, bones would survive impact better than meat.
+	hits = 4 // Bone is pretty hard.
+	meteorsound = 'sound/magic/RATTLEMEBONES2.ogg' // ME BONES
+	threat = 5
+	
+/obj/effect/meteor/meaty/bone/make_debris()
+	for(var/throws = dropamt, throws > 0, throws--)
+		var/thing_to_spawn = pick(meteordrop)
+		var/atom/thing = new thing_to_spawn(get_turf(src))
+		for(var/j = 1, j <= rand(1, 3), j++)
+			step(thing, pick(NORTH,SOUTH,EAST,WEST))
+
+//Long ago in a distant fairy land, there was a man known far and wide...
+/obj/effect/meteor/meaty/meatball_man
+	name = "meatball man"
+	icon = 'icons/effects/512x512.dmi'
+	icon_state = "meatball_man"
+	pixel_x = -240
+	pixel_y = -240
+	desc = "Oh, what a feat! He's one with the meat!"
+	hits = INFINITY // You cannot stop the Meatball Man.
+	lifetime = 0 // Meatball Man is eternal.
+	spin = FALSE
 	hitpwr = 1
 	heavy = 1
-	meteorsound = 'sound/effects/bamf.ogg'
-	meteordrop = list(/obj/item/stack/ore/plasma)
-	threat = 50
-
-/obj/effect/meteor/tunguska/Move()
+	meteorsound = 'sound/effects/blobattack.ogg'
+	threat = 200
+	move_delay = 40 // Meatball Man never rushes, for all will become Meat in time.
+	var/meaty_range = 14
+	var/eaty_range = 7
```

Comment:
so you're gonna get ex_act()ed as soon as the meatball man comes on screen? boy, that'll feel like a fair death

Reply:
It moves extremely slowly, gives a audio cue, and produces huge quantities of meat in front of its path. But good point, I'll try and figure out how to make it render from farther out.

Reply:
Also strictly speaking this is pretty rare and unlikely, and it's precedent that sudden-but-rare sources of danger/death are fine. See: Immovable rod, spontaneous diseases/brain trauma, other meteors(Tunguska can penetrate a entire department, gib you, and then maxcap your gibs), etc.

Reply:
(Also, even if you can't see the floor you'll notice something inexplicably destroying it in a slowly-approaching wave. If _that_ isn't a hint that you need to move, what is?)

Reply:
Because with singulos, you still have time even after you see the edge of the singulo on your screen to RUN. With the meatball man, you're dead upon him coming into visual range (if he happens to move into your viewing range). It's like ramming an avatar of Nar'Sie into the station.

Reply:
If you can _see_ a stage 5 singularity (which it usually is, because it reaches stage 5 very quickly), you're not only so close that you're long since in space and getting dragged in along with the rest of the room, but you're almost (2 tiles away) within the range where it eats you outright _while getting rapidly pulled in while in space with nothing but other infalling debris to push off of._ And there's a fair chance it's moving towards you too, to add insult to injury.

The time when you can practically run from a singularity is when you first notice that part of the room you're in is being dismantled and sucked away. In other words, when the _edge_ of the pull radius is, not even in range of you, but _in range of your view range_. Because you have to be blind and deaf to not notice half of the room you're in suddenly disappear amid smashing and breaking noises. If you can _SEE_ the singularity itself, you are almost certainly about to die.


This is, for the record, more warning than meteors are precedented to give you. Meteors can gib you within a quarter second of you hearing/seeing them if you're anywhere near the edge of the station, especially for catastrophic meteor swarms where occasionally a maxcap will just fly 20-40 blocks into the station and detonate a entire department within a span of two seconds total.

This guy, meanwhile:
1: Warns you with a comparable range to a singularity with a very obvious but otherwise non-inhibiting warning in the form of ludicrous gibs.
2: Moves slow as fuck.
3: _Sends a completely distinct audio cue to the entire server._

And finally, fucks sake, its rare and it's a _meme_. It's fine, if there's anything I thought people would be worried about, it's that it'd be too destructive when it spawned.